DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Features/PolylineFeatureToSpeckleConverter.cs

```diff
@@ -1,40 +1,26 @@
 using ArcGIS.Core.Geometry;
 using Speckle.Converters.Common.Objects;
 using Speckle.Core.Models;
-using Speckle.Converters.Common;
-using ArcGIS.Desktop.Mapping;
-using ArcMapPoint = ArcGIS.Core.Geometry.MapPoint;
 
 namespace Speckle.Converters.ArcGIS3.Features;
 
-[NameAndRankValue(nameof(Polyline), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
-public class PolyineFeatureToSpeckleConverter : IHostObjectToSpeckleConversion, IRawConversion<Polyline, Base>
+public class PolyineFeatureToSpeckleConverter : IRawConversion<Polyline, List<Base>>
```

Comment:
I'm not sure about returning so many List<Base>, it is somewhat unspecific. I would much rather have a clear picture of what is this returning, which I'm assuming would be a List<SOG.Polyline>?